Search Page

ayse8888 opened the pull request:
Hi All
I built the Search Page. #35 
For some reason, I observed some styles are broken as attached in the cards. Also in home page and about us page modals.
I think there are some duplicate class / id names and they override each other.
I believe we can fix this when everything is merged
Please correct me if I m wrong @halitbatur 

![image](https://user-images.githubusercontent.com/65858467/131035206-f8ba428b-eee8-423a-b4c8-c0854165f2b8.png)
![image](https://user-images.githubusercontent.com/65858467/131035233-fdf45133-cdb6-40d6-b8d3-8abc6464c57d.png)

ayse8888 committed:
Build:Add SearchPage

halitbatur commented on src/App.js:
```diff
@@ -14,7 +15,8 @@ function App() {
       <Route path="/resources" component={ResourcesPage} />
-      <Route exact path="/about" component={AboutUs} />
+      <Route path="/about" component={AboutUs} />
+      <Route path="/search" component={SearchPage} />
```
why does this and aboutus does not have the option exact ?

ayse8888 replied:
sorry Halit, i think i did not notice that.  i fixed both with exact path now

ayse8888 committed:
exact path issue fixed

ayse8888 committed:
fixed form issues and updated some css

ayse8888 commented:
Hi,
I needed to make some updates on some components so I used this branch.
1. in the search component, I added email.js to send the data from adoption form to my email address
![image](https://user-images.githubusercontent.com/65858467/131264847-b5072497-0585-47e2-84d9-d544ab89072f.png)


2. i changed some css in the search component and in the latest guests because everyone used duplicate/same class/id names and I changed mine so the css is not broken any more.
![image](https://user-images.githubusercontent.com/65858467/131264833-b16a0965-57fb-4016-82bb-fb33286610f4.png)

3. in the about us page, i added mustafa's picture

halitbatur commented on src/components/Search/Search.css:
```diff
@@ -6,8 +6,8 @@
 
-.adoptButton,
-.adoptButton:hover {
+#adoptButton,
```
why are we using ids here could we use className

ayse8888 replied:
hi halit, yeah actually i had to. bcoz of the some duplicate classnames from other components, they override each other and break the css. it was like in the first picture. all css were broken. and i changed the class and ids to fix it. this one was also a classname before but i changed it to id for that reason

ayse8888 committed:
conflict fixed

ayse8888 committed:
conflict fixed

ayse8888 committed:
yarn run fix
